styleguide/Pages/ContactTablesNoTOC.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace Styleguide\Pages;
+
+class ContactTablesNoTOC extends Page
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function getPageData()
+    {
+        return app('Factories\Page')->create(1, true, [
+            'data' => [
+                'profile_group_id' => '0|1|2|3',
+                'table_of_contents' => 'hide',
+            ],
+            'page' => [
+                'controller' => 'ContactTableController',
+                'title' => 'Contact tables, no table of contents',
+                'id' => 101114,
+                'content' => [],
```

Comment:
Can the page content for this styleguide page explain how to use it to anyone viewing the styleguide instead of requiring a developer to dive into the code to add the configuration to the CMS to use it.